build.sh

```diff
@@ -19,7 +19,9 @@ mkdir /tmp/specifications && cp content/specifications/_index.md /tmp/specificat
 rm -rf content/specifications && rm -rf static/specifications
 
 echo "Step 2: Clone specifications git repository"
-git clone https://github.com/jakartaee/specifications.git content/specifications
```

Comment:
@linkfang I assume this was left in your PR by mistake?

Reply:
Ah, yes, good catch!